apps.yaml

```diff
@@ -12,12 +12,9 @@ aiidalab-optimade:
   - utilities
   logo: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/232f11d39076b328e07c8bb6928c3a6847523f6c/img/optimade-no-text-transparent-bg.png
   metadata:
-    # TODO: Switch URL to "develop" branch after testing.
-    $ref: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/integrate-aiidalab-app/.aiidalab/metadata.yaml
+    $ref: https://raw.githubusercontent.com/CasperWA/voila-optimade-client/develop/.aiidalab/metadata.yaml
   releases:
-    # TODO: Update branch after testing.
-  - "git+https://github.com/CasperWA/voila-optimade-client.git@integrate-aiidalab-app:2021.8.4rc0+aiidalab^.."
-  - "git+https://github.com/aiidalab/aiidalab-optimade.git@:"
```

Comment:
Was removing the old releases intentionally? If so, I'd appreciate if we keep this for now, since we need at least one "standard" git branch to maintain backwards compatibility with the old `apps_meta.json`. This will not be needed anymore in the near future.

Reply:
To be clear, I am specifically referring of the removal of line 20.